multi-node/aws/pkg/config/config.go

```diff
@@ -116,7 +120,15 @@ const (
 
 func (c Cluster) Config() (*Config, error) {
 	config := Config{Cluster: c}
-	config.ETCDEndpoints = fmt.Sprintf("http://%s:2379", c.ControllerIP)
+	etcdEndpoints := strings.Split(c.EtcdEndpoints, ",")
+	for i := range etcdEndpoints {
+		_, err := url.Parse(etcdEndpoints[i])
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
Somewhat nit-picky but I'd prefer if this loop didn't rely on the index and did something like `for _, endpoint := range etcdEndpoints` as @chancez did in his example.